include/deal.II/lac/petsc_precondition.h

```diff
@@ -562,11 +562,12 @@ namespace PETScWrappers
 
 
   /**
-   * A class that implements the interface to use the PETSc LU preconditioner (@p PCLU).
+   * A class that implements the interface to use the PETSc LU preconditioner
+   * (@p PCLU). This usually performs an exact factorization of the matrix, so
+   * it is not necessary to wrap it in a linear solver.
```

Comment:
But it *is* in a class that's named `Preconditioner...`. Is what you want to say here something along the lines of "Unlike classes that implement an ILU, this class actually performs an exact factorization of the matrix. Consequently, it can be used as the perfect preconditioner in an iterative scheme, but it can also be used as a stand-alone solver for the linear problem."?

Of course, the class does not have an interface that allows using it as a stand-alone solver, so the second sentence above is not actually correct.

Reply:
> Of course, the class does not have an interface that allows using it as a stand-alone solver,

I thought you can do ``pre.vmult(dst, src)``.

Reply:
You can also use SolverPreOnly

Reply:
Ah, I see. All of these might make for a nice additional sentence :-)